3_visualize/log/daily_timeseries_summary.csv

```diff
@@ -0,0 +1,112 @@
+site_no,n_obs,mean,sd
+01411400,2,141,0
+01411500,3158,77.06188511583625,8.128900673491826
```

Comment:
Can probably round to 6 decimal places for mean and sd. There may be some numerical precision differences between users, but out to 6 places should be the same

Reply:
Great point, I'll make the rounding change in the `summarize_timeseries.R` file.

Reply:
Rounding changes made in new commit `2ff9b1f`